torchci/lib/bot/ciflowPushTrigger.ts

```diff
@@ -150,19 +150,24 @@ async function handleLabelEvent(
     );
     return;
   }
-
   if (!valid_labels.includes(label)) {
     let body = `Unknown label \`${label}\`.\n Currently recognized labels are\n`;
     valid_labels.forEach((l: string) => {
       body += ` - \`${l}\`\n`;
     });
+    let is_first_time_contributor = await isFirstTimeContributor(context, context.payload.pull_request.user.login);
```

Comment:
During the discussion, we talked about gating this logic to prevent unknown first-time contributors from doing this, so this is ok.  But after thinking about this a bit more, I think it's better to check if the contributors have [write access to PyTorch](https://github.com/pytorch/test-infra/blob/e205f9aa5119ab60ef8f6b7ae0d8d3b84a8c0bcc/torchci/lib/bot/utils.ts#L170-L176).  That's stricter than the first-time contributor check.  Doing a write access check makes more sense to me because that's more align with what is done here, "writing" a new label.  Thoughts?

Reply:
sounds reasonable to me, I was considering doing the same.